build/dsc/manager.rb

```diff
@@ -214,6 +214,41 @@ def clean_dmtf_mofs
       clean_folder([@dmtf_mof_folder])
     end
 
+    def format_embeddedtype_all_values(property)
```

Comment:
This isn't quite right as it fails when `manager.rb` has the extraneous types omitted.